torch/_functorch/_aot_autograd/jit_compile_runtime_wrappers.py

```diff
@@ -549,9 +549,15 @@ def prepare_for_partitioner(mod, num_primals, num_fw_outputs):
                     joint_hop_gm, num_fw_inputs, num_fw_outputs
                 )
 
+                # TODO: invoke_subgraph should track which of its inputs static indices
```

Comment:
fyi @anijain2305 I didn't fix this here, but we might have to do some extra work for invoke_subgraph to make sure to tell the partitioner which inputs to an invoke_subgraph call are "static". I'll link this into the doc of followups you added if that sounds ok